lawnchair/src/app/lawnchair/util/OptionalUtil.kt

```diff
@@ -1,9 +0,0 @@
-package app.lawnchair.util
-
-import java.util.*
-
-val <T> Optional<T>.value: T?
-    get() {
-        if (!isPresent) return null
-        return get()
-    }
```

Comment:
https://kotlinlang.org/docs/whatsnew17.html#new-experimental-extension-functions-for-java-optionals